src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,47 @@
 'use strict';
 
+const HEALTH = 100;
+const BITE_POWER = 50;
+
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = HEALTH;
+    Animal.aliveAnimalsList.push(this);
+  }
+
+  static get alive() {
+    return Animal.aliveAnimalsList.filter(animal => animal.health > 0);
+  }
```

Comment:
You can change this to method `die` which will filter static `alive` array. And call this method on `victim` in method `bite` of `Carnivore`